package.json

```diff
@@ -62,7 +62,7 @@
     "ms": "^2.1.3",
     "secure-json-parse": "^2.4.0",
     "tslib": "^2.4.0",
-    "undici": "^5.1.1"
+    "undici": "^5.5.1"
```

Comment:
__note:__ let me know if I need to bump version of the `@elastic/transport` package itself as well.